tests/test_interface.rb

```diff
@@ -1350,20 +1325,18 @@ def test_interface_ipv4_all_interfaces
     rescue Minitest::Assertion
       # clean up before failing
       config(*cfg)
-      InterfaceChannelGroup.new(interfaces[1]).channel_group = false
+      interface_ethernet_default(interfaces[1])
```

Comment:
Did you check this change on XR? IIRC this code was needed due to an issue where 'default interface Gig0/0/0/0' wasn't removing 'bundle id 10' from the interface.